impact_stack/rest/__init__.py

```diff
@@ -81,6 +82,7 @@ def __init__(self, base_url, api_key):
         """Create a new client factory instance."""
         self.base_url = base_url
         self.client_classes = collections.defaultdict(lambda: self.DEFAULT_CLASS)
+        self.timeouts = collections.defaultdict(lambda: self.DEFAULT_TIMEOUT)
```

Comment:
Why can/should `DEFAULT_CLASS` and `DEFAULT_TIMEOUT` not be part of the app config? :thinking:

Reply:
I’ve made another attempt at making configuration more comfortable. It now allows for:

```python
IMPACT_STACK_API_CLIENT_DEFAULTS={"class": CustomClient, "timeout": 3}
IMPACT_STACK_API_CLIENT_CONFIG={"slow-service": {"timeout": 60}}
```

Reply:
Thanks, I find the new version more intuitive to use :)